W-14244909 Adding rename folder (#306)
mulesoft/docs-rpa: rpa-builder/6.5/modules/ROOT/pages/toolbox-folder-operations-rename-folder.adoc

= Rename Folder 

Renames an existing folder. 

== Properties 

* *Common* 
+
See xref:common-properties.adoc[Common Properties] for a description.  

* *Folder Operations Settings* 
** *Directory Path* 
+

> Path to the directory where to check the existence of the folder. 

Reviewer: ```suggestion
Path to the existing directory which will be renamed. 
```
